### PR TITLE
[Growth] Query events from owner and proxies

### PR DESCRIPTION
**Pull request by franckc**
### Description:

With the launch of meta-txn, the listener back-end will be recording GrowthEvents (e.g. Identity updates, Listing Sold/Purchased, etc...) under the address of either the wallet (in case user paid for gas) or the proxy (in case relayer paid for gas).

This PR updates the rules logic so that it loads events recorder under the wallet and any proxies owned by it.

The DApp always queries the growth back-end using the wallet address regardless of the wallet having aproxy or not. As a result, the user should see rewards for actions performed by either wallet or proxies.

Note: the fraud and payout pipeline needs to be updated to take into account proxies. I'll handle that as a separate PR.

### Checklist:

- [ ] Test your work and double-check to confirm that you didn't break anything
- [ ] [Wrap any new text/strings with `fbt`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#wrapping-text) for translation
- [ ] If there are any new/changed strings to translate, [`npm run translate`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#integrating-translations)
- [ ] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/docs)

### CODE REVIEW

**Comment by franckc** [comment]
CC @nick as a FYI

File: infra/growth/src/resources/rules.js [hunk 1]
```diff
@@ -85,9 +90,14 @@ class CampaignRules {
...
-    const whereClause = {
-      ethAddress: ethAddress.toLowerCase()
-    }
+
+    // Load any proxy associated with the wallet address.
+    const ownerAddress = ethAddress.toLowerCase()
+    const proxies = await db.Proxy.findAll({ where: { ownerAddress } })
+
+    // Query events from wallet and proxy(ies).
+    const addresses = [ownerAddress, ...proxies.map(proxy => proxy.address)]
```
Comment (sparrowDom) [inline infra/growth/src/resources/rules.js]: Should we call `toLowerCase()` on proxy addresses as well?

Comment (franckc) [reply infra/growth/src/resources/rules.js]: The proxy table already stores addresses in lower case so that should not be necessary.

**Comment by franckc** [comment]
Thanks @sparrowDom for the thorough review and quick turnaround ! Going to merge this to master and will deploy today...

**Pull request merged**